stdlib/InteractiveUtils/test/runtests.jl

```diff
@@ -249,10 +249,10 @@ end
         @test occursin("Environment:", ver)
     end
     let exename = `$(Base.julia_cmd()) --startup-file=no`
-        @test !occursin("Environment:", read(setenv(`$exename -e 'using InteractiveUtils; versioninfo()'`,
-                                                    String[]), String))
-        @test  occursin("Environment:", read(setenv(`$exename -e 'using InteractiveUtils; versioninfo()'`,
-                                                    String["JULIA_CPU_THREADS=1"]), String))
+        ecpy = copy(ENV)
+        @test !occursin("Environment:", read(setenv(`$exename -e 'using InteractiveUtils; versioninfo()'`, ecpy), String))
```

Comment:
Does this actually pass? I'm assuming this is checking that `versioninfo` doesn't print the environment if none is passed.